src/app/dashboard/Missions/CreateMission.js

```diff
@@ -0,0 +1,482 @@
+  const [comment, setComment] = useState("");
+  const [foodBoxArray, setFoodBoxArray] = useState([]);
+  // master mission object
+  const [newMissionObject, setNewMissionObject] = useState({});
+  // master submit
+  async function submitMission(e) {
+    e.preventDefault();
+    setNewMissionObject({
+      missionId: uuidv4(),
+      createdAt: new Date(),
+      createdBy: "organizer", // make dynamic
+      missionType: missionType,
+      associatedItems: items,
+      fund: missionFund,
+      recipient: recipient,
+      volunteer: volunteer,
+      dropOff: dropOff,
+      pickUp: pickUp,
+      comment: comment,
+    });
+    try {
+      const createdMission = await Mission.create(newMissionObject);
+      console.log(createdMission);
+    } catch (error) {
+      console.log(error);
+    }
+  }
+  // Functions for changing the states
+  const handleChangeMissionType = (event) => {
+    setMissionType(event.target.value);
+  };
+
+  const handleChangeFund = (event) => {
+    setMissionFund(event.target.value);
+  };
+
+  const handleRecipient = (event) => {
+    setRecipient({ ...recipient, [event.target.name]: event.target.value });
+  };
+
+  const handleVolunteer = (event) => {
+    setVolunteer({ ...volunteer, [event.target.name]: event.target.value });
+  };
+
+  const handleDrop = (event) => {
+    setDropOff({ ...dropOff, [event.target.name]: event.target.value });
+  };
+
+  const handlePickUp = (event) => {
+    setPickUp({ ...pickUp, [event.target.name]: event.target.value });
+  };
+
+  const handleComment = (event) => {
+    setComment(event.target.value);
+  };
+
+  const handleItems = (event) => {
+    setItems({ ...items, [event.target.name]: event.target.value });
+    console.log(items);
+  };
+
+  function addToFBCount() {
+    setFoodBoxArray([...foodBoxArray, foodBoxArray.length]);
+  }
+
+  return (
+    <Container maxWidth="sm">
+      <Typography className={classes.addMissionContainer} component="div">
+        <div className={classes.title}>
+          <AnnouncementIcon className={classes.icon} />
+          <Typography className={classes.titleHeader}>Add New Mission</Typography>
+        </div>
+        <Divider variant="middle" />
+        <Typography className={classes.inputHeader}>Mission Type</Typography>
+        <FormControl className={classes.form}>
+          <InputLabel
+            variant="outlined"
+            id="missionType"
+            shrink={true}
+            className={classes.inputLabel}
+          >
+            Select Mission Type
+          </InputLabel>
+          <Select
+            labelid="missionType"
+            variant="outlined"
+            className={classes.select}
+            value={missionType}
+            onChange={handleChangeMissionType}
+          >
+            <MenuItem className={classes.menuItem} value="food-box">
+              Food box
+            </MenuItem>
+          </Select>
+        </FormControl>
+        {missionType !== null ? (
```

Comment:
@thirdeyeclub You don't need to use a ternary if you actually only check for one condition. You can use `&&`. See https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator